pypeit/spectrographs/not_alfosc.py

```diff
@@ -28,7 +28,7 @@ class NOTALFOSCSpectrograph(spectrograph.Spectrograph):
     url = 'https://www.not.iac.es/instruments/alfosc/'
     header_name = 'ALFOSC_FASU'
     supported = True
-    comment = 'Grisms 4, 19'
+    comment = 'Grisms 4, 8, 17, 19'
```

Comment:
Maybe with the comment here you could add "For use with the horizontal slit mode only."